examples/lines_bars_and_markers/timeline.py

```diff
@@ -53,10 +53,10 @@
     dates = [datetime.strptime(d, "%Y-%m-%d") for d in dates]
 
 ##############################################################################
-# Next, we'll create a `~.Axes.stem` plot with some variation in levels as to
-# distinguish even close-by events. In contrast to a usual stem plot, we will
-# shift the markers to the baseline for visual emphasis on the one-dimensional
-# nature of the time line.
+# Next, we'll create a stem plot with some variation in levels as to
```

Comment:
I wouldn't call it "stem plot" then.

Reply:
I know, but I couldn't really think of a better name...

Reply:
IMHO the whole section could be rewritten to better descibe the intention and how we use the matplotlib functions to achieve that. But let's not make perfect the enemy of the good.